public/employers/levi-strauss.yml

```diff
@@ -0,0 +1,58 @@
+name: Levi Strauss & Co.
+aliases:
+  - Levi's
+shortName: Levi Strauss
+ticker: LEVI
+wiki: Levi Strauss & Co.
+image: levi-strauss.svg#a8112e
+officialWebsite: https://www.levistrauss.com/levis-history/
+industries:
+  - apparel
+  - retail
+  - manufacturing
+location:
+  city: San Francisco
+  state: CA
+  country: us
+  international: true
+  wiki: San Francisco
+employeesBefore:
+  type: approximately
+  upperBound: 15100
+  year: 2018
+summary: Levi Strauss closed its stores as required and initially committed to paying furloughed employees, but later cut their pay entirely. Meanwhile, it issued dividends to shareholders while also reducing their CEO's salary.
+citations:
+  - summary: Levi Strauss issued a dividend of $32 million USD despite laying off a large number of employees.
+    positivity: -1
+    type: publication
+    sources:
+      - source: The Washington Post
+        link: https://www.washingtonpost.com/business/2020/05/05/dividends-layoffs-coronavirus/
+        date: 2020-05-05T14:51:26Z
+  - summary: Levi Strauss's CEO took a pay cut of 50%.
```

Comment:
Shouldn't this be Strauss' ?

I also think you can shorten it to just Levi.

Reply:
It's actually fine this way (both `s'` and `s's` are accepted) but I'll just reword to remove the issue entirely.